@ecomplus/storefront-template/template/js/index.js

```diff
@@ -9,6 +9,7 @@ import './lib/menu'
 import './lib/search'
 import './lib/shopping-cart'
 import './lib/persist-utm'
+import './lib/theme'
```

Comment:
Acho que isso vai ter que ser importado antes

Reply:
Ele deve ter que ser importado antes do https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-template/template/js/lib/config.js

Reply:
Que por sua vez é importado no https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-template/template/js/lib/ecom.js

Reply:
Então o `lib/theme` deve ter que ser importado no index antes de https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-template/template/js/index.js#L2

Reply:
Pro `window._theme` já estar definido quando for setar o `window.storefront`, faz sentido?